data/microos/utt-alp.repo

```diff
@@ -0,0 +1,7 @@
+[openSUSE_Factory_utt]
+name=openSUSE:Factory:utt
+type=rpm-md
+baseurl=https://download.opensuse.org/repositories/openSUSE:/Leap:/Micro:/5.1:/Update/standard/
```

Comment:
Not this one `https://download.opensuse.org/repositories/devel:/LEO/standard/` ?

Reply:
No, because 2 things:
- This repo doesn't include UTT packages
- This repo addition is not for all the updates, it's meant ony for UTT packages. If I add the repo you suggest, it will "override" the packages that we have synced from local in openQA.

Maybe we can ask someone to make some opensuse generic repo in OBS for UTT only... or rename this Leap Micro 5.1 to something more generic we can use in ALP and Leap Micro... I only found this one which works...  
Tumbleweed already contains UTT packages in the Factory repo...

Reply:
Already asked, we might be using the same we use for Leap Micro, waiting now.

Reply:
Already asked, we might be using the same we use for Leap Micro, waiting now.